### ISSUE TITLE
Need to ignore files/directories during index generation

### ISSUE THREAD

**Issue by LukeMauldin** [m2mdas/phpcomplete-extended#31]
I am getting errors when creating the index from external packages like PHPExcel.  The file jpgraph.php has this line:
require_once(PHPExcel_Settings::getChartRendererPath().'/jpgraph.php');

When phpcomplete-extended is running, the 'getChartRendererPath()' is blank and so the require_once fails due to being unable to find jpgraph.php.  I need to be able to exclude certain files/directories/packages from index generation so index generation can complete successfully.  It is better than the index be incomplete (ie  not work on certain files/directories) than not to be able to use it at all.

**Comment by m2mdas** [comment]
It is a nice feature request. Hoping to implement it on this weekend

**Comment by ghost** [comment]
Yeah, i have a problem with a vendor too, this would be a very nice way to handle this.

**Comment by aewing** [comment]
+1

**Comment by nce** [comment]
ran into the same problem today involving codeception files in my `vendor` directory.

**Comment by jdwil** [comment]
+1

**Comment by willdurney** [comment]
+1

**Comment by mohammed-elhakim-sociomantic** [comment]
+1

**Comment by andrea-imparato-sociomantic** [comment]
+1

**Comment by idchlife** [comment]
So, how to overcome this problem? Plugin always asks me to build index and always has problem with some codeception etc files in vendor.
